Product/Production/Adapters/General/CONNECTDirectConfig/src/main/java/gov/hhs/fha/nhinc/directconfig/dao/impl/CertificateDaoImpl.java

```diff
@@ -74,18 +74,18 @@
 @Repository
 public class CertificateDaoImpl implements CertificateDao {
 
-    private static final Log log = LogFactory.getLog(CertificateDaoImpl.class);
+    private static final Log LOG = LogFactory.getLog(CertificateDaoImpl.class);
 
     /**
      * {@inheritDoc}
      */
     @Override
-    @SuppressWarnings({ "unchecked" })
+    @SuppressWarnings({"unchecked"})
```

Comment:
Eventually these will be removed, but anything we inherited from Nhin Direct will stay intact for now.